<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 25 12pt &quot;Roboto&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="avatar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/avatar.png</pixmap>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabSidebar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>255</y>
      <width>241</width>
      <height>301</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(228, 238, 242);
border: none;
border-top: #FFF;</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="firstTab">
     <attribute name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/online.png</normaloff>:/images/online.png</iconset>
     </attribute>
     <attribute name="title">
      <string>EN LIGNE</string>
     </attribute>
     <widget class="QListWidget" name="onlineList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-right: 1px solid #000;
gridline-color: rgb(0, 0, 0);
padding-top: 15px;
</string>
      </property>
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::NoDragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="movement">
       <enum>QListView::Static</enum>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="layoutMode">
       <enum>QListView::SinglePass</enum>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
      <property name="modelColumn">
       <number>0</number>
      </property>
      <property name="uniformItemSizes">
       <bool>false</bool>
      </property>
      <property name="selectionRectVisible">
       <bool>true</bool>
      </property>
      <property name="currentRow">
       <number>-1</number>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Babel Echo Test</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Julien Loir</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nicolas Lequain</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jérémy Mathon</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Martin Kinoo</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="secondTab">
     <attribute name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/friends.png</normaloff>:/images/friends.png</iconset>
     </attribute>
     <attribute name="title">
      <string>AMIS</string>
     </attribute>
     <widget class="QListWidget" name="friendList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-right: 1px solid #000;
padding-top: 15px;</string>
      </property>
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <property name="text">
        <string>Aucun ami</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>511</y>
      <width>88</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ForbiddenCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #B6B6B6;
background-color: rgb(228, 238, 242);
text-transform: uppercase;</string>
    </property>
    <property name="text">
     <string>Envoyer</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/plane.png</normaloff>:/images/plane.png</iconset>
    </property>
   </widget>
   <widget class="QListWidget" name="historyList">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>116</y>
      <width>531</width>
      <height>371</height>
     </rect>
    </property>
    <property name="tabKeyNavigation">
     <bool>true</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>Hey salut mec, ça va ?</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Trkl et toi ?</string>
     </property>
     <property name="textAlignment">
      <set>AlignRight|AlignVCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ouais, oklm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Appel (durée 04:35)</string>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="userLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 20pt &quot;Roboto&quot;;
color: #12A5F4;</string>
    </property>
    <property name="text">
     <string>Babel Echo Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="callButton">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>40</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ForbiddenCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #B6B6B6;
color: #FFFFFF;
background-color: #4CAF50;
border-radius: 25px;
text-transform: uppercase;</string>
    </property>
    <property name="text">
     <string>Appeler</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/phone.png</normaloff>:/images/phone.png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="mainLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFF;
font-size: 24px;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="mainOnline">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>66</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>en ligne</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
border-right: 1px solid #000;
background-color: #12A5F4;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>66</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/images/ok.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="videoButton">
    <property name="geometry">
     <rect>
      <x>588</x>
      <y>40</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ForbiddenCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #B6B6B6;
background-color: #009688;
color: #FFFFFF;
text-transform: uppercase;</string>
    </property>
    <property name="text">
     <string>Vidéo</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/camera.png</normaloff>:/images/camera.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="friendButton">
    <property name="geometry">
     <rect>
      <x>689</x>
      <y>40</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ForbiddenCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #B6B6B6;
background-color: #FFC107;
color: #FFFFFF;
text-transform: uppercase;</string>
    </property>
    <property name="text">
     <string>Ami</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/star.png</normaloff>:/images/star.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>121</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/glass.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="researchLine">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>117</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
font-weight: bold;
font-family: 'Robot';
background-color: rgb(228, 238, 242);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Rechercher</string>
    </property>
   </widget>
   <widget class="QListWidget" name="sidebarList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(228, 238, 242);
font: 14pt &quot;Roboto&quot;;
border: none;
border-right: 1px solid #000;
border-bottom: 1px solid #12A5F4;
</string>
    </property>
    <property name="tabKeyNavigation">
     <bool>true</bool>
    </property>
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
    <item>
     <property name="text">
      <string>Accueil</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/home.png</normaloff>:/images/home.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Récents appels</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/phone.png</normaloff>:/images/phone.png</iconset>
     </property>
    </item>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(228, 238, 242);
border: none;
border-right: 1px solid #000;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>228</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/images/arrow.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="userOnline">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>58</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>en ligne</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>65</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/ok.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>-20</y>
      <width>591</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFFFFF;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>581</width>
       <height>101</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="sendText">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>500</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>Ecrivez un message à votre contact...</string>
    </property>
   </widget>
   <zorder>frame_3</zorder>
   <zorder>frame</zorder>
   <zorder>tabSidebar</zorder>
   <zorder>sendButton</zorder>
   <zorder>historyList</zorder>
   <zorder>userLabel</zorder>
   <zorder>callButton</zorder>
   <zorder>frame_2</zorder>
   <zorder>avatar</zorder>
   <zorder>mainLabel</zorder>
   <zorder>mainOnline</zorder>
   <zorder>videoButton</zorder>
   <zorder>friendButton</zorder>
   <zorder>label_8</zorder>
   <zorder>researchLine</zorder>
   <zorder>sidebarList</zorder>
   <zorder>userOnline</zorder>
   <zorder>label_7</zorder>
   <zorder>sendText</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBabel">
    <property name="title">
     <string>Babel</string>
    </property>
   </widget>
   <addaction name="menuBabel"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
